website/blog/2021-11-24-2.5.0.md

```diff
@@ -0,0 +1,341 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.5: TypeScript 4.5 and MDX v2 comment syntax!"
+---
+
+This release adds support for [TypeScript 4.5](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/)'s new syntax and [MDX v2](https://mdxjs.com/blog/v2/) comment syntax!
+
+If you enjoy Prettier and would like to support our work, consider sponsoring us directly via [our OpenCollective](https://opencollective.com/prettier) or by sponsoring the projects we depend on, including [typescript-eslint](https://opencollective.com/typescript-eslint), [remark](https://opencollective.com/unified), and [Babel](https://opencollective.com/babel).
+
+<!--truncate-->
+
+## Highlights
+
+### TypeScript
+
+#### Avoid extra offset in arrow function body when using long types ([#11515](https://github.com/prettier/prettier/pull/11515) by [@kachkaev](https://github.com/kachkaev) and [@thorn0](https://github.com/thorn0))
+
+Starting with Prettier 2.3.0, type declarations in arrow functions could affect function body offset. Changing the length of the type annotation could produce large diffs and thus increased the chance of git conflicts. To prevent this, function body offset was stabilized.<br>
+**Note**: This change may affect a large number of lines in your codebase.
+
+<!-- prettier-ignore -->
+```tsx
+// Input
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> = ({ x, y }) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+
+// Prettier 2.2 and below
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> = ({
+  x,
+  y,
+}) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+
+// Prettier 2.4
+const MyComponentWithLongName: React.VoidFunctionComponent<MyComponentWithLongNameProps> =
+  ({ x, y }) => {
+    const a = useA();
+    return <div>{x + y + a}</div>;
+  };
+
+// Prettier 2.5
+const MyComponentWithLongName: React.VoidFunctionComponent<
+  MyComponentWithLongNameProps
+> = ({ x, y }) => {
+  const a = useA();
+  return <div>{x + y + a}</div>;
+};
+```
+
+#### Support TypeScript 4.5 ([#11721](https://github.com/prettier/prettier/pull/11721), [#11723](https://github.com/prettier/prettier/pull/11723), [#11813](https://github.com/prettier/prettier/pull/11813) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+We’ve added support for TypeScript 4.5's new syntax features:
```

Comment:
A veeeeery minor thing that’s not even worth a push if there is nothing else to change. I’ve noticed the usage of `'` (typewriter apostrophe) instead of `’` ([typographic apostrophe](https://en.wikipedia.org/wiki/Apostrophe#Typographic_form)) in a couple of places. Could be worth searching for `'s` and replacing with`’s` for consistency with most text. Note that `'` is used in code snippets so we should be careful 😁